resources/views/site/staff.blade.php

```diff
@@ -72,11 +72,12 @@ class="img-responsive center-block profile-picture"/>
                             </tr>
                             <tr>
                                 <td>Aerodrome Operations Coordinator</td>
-                                <td>Vacant</td>
+                                <td>Chad Byworth</td>
                             </tr>
                             <tr>
-                                <td>Sector File Coordinator</td>
-                                <td>Vacant</td>
+                                <td>Sector File Coordinators</td>
+                                <td>Luke Brown<br>
+								Thomas Mills</td>
```

Comment:
```html
<tr>
    <td>Sector File Coordinator</td>
    <td>Luke Brown</td>
</tr>
<tr>
    <td>Sector File Coordinator</td>
    <td>Thomas Mills</td>
</tr>
```
Each person should have their own row to keep it consistent with other areas of this page.

Reply:
Fixed!